roles/default/kiali-deploy/defaults/main.yml

```
@@ -90,6 +90,8 @@ kiali_defaults:
 
   external_services:
     custom_dashboards:
+      discovery_enabled: "auto"
+      discovery_auto_threshold: 10
```

All peer settings in the operator config files are sorted alphabetically. Sort these alphabetically:

```suggestion
      discovery_auto_threshold: 10
      discovery_enabled: "auto"
```